lgsm/functions/install_server_files.sh

```diff
@@ -62,6 +62,10 @@ fn_install_server_files(){
 		remote_fileurl="https://files.sa-mp.com/samp037svr_R2-1.tar.gz"; local_filedir="${tmpdir}"; local_filename="samp037svr_R2-1.tar.gz"; chmodx="nochmodx" run="norun"; force="noforce"; md5="93705e165550c97484678236749198a4"
 	elif [ "${shortname}" == "zmr" ]; then
 		remote_fileurl="http://linuxgsm.download/ZombieMasterReborn/zombie_master_reborn_b5_2.tar.xz"; local_filedir="${tmpdir}"; local_filename="zombie_master_reborn_b5_2.tar.xz"; chmodx="nochmodx" run="norun"; force="noforce"; md5="4b9b9832e863d03981a40c26065792a6"
+	else
+		fn_print_error "Installing ${gamename} Server failed, missing default configuration"
```

Comment:
LinuxGSM would fail long before getting here is $shortname was was not available

Reply:
bdserver will run into this without the fix from following pr, makes it more safe to use here.
https://github.com/jusito/LinuxGSM/blob/fix/bdserver/lgsm/functions/install_server_files.sh#L13-L16